src/database/models/activities.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
+	const Activity = sequelize.define(
+		'Activity',
+		{
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: DataTypes.INTEGER,
+			},
+			user_id: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			action_id: {
+				allowNull: false,
+				type: DataTypes.INTEGER,
+			},
+			object_id: {
```

Comment:
@priyanka-TL should object id be indexed, because for a given resource people might want all the activities